tool/cli.ts

```diff
@@ -365,11 +365,40 @@ program
             default: true,
           });
       if (run) {
-        const removed = Document.remove(slug, locale, {
+        const deletedDocs = Document.remove(slug, locale, {
           recursive,
           redirect,
         });
-        console.log(chalk.green(`Moved ${removed.length} documents.`));
+        console.log(chalk.green(`Deleted ${deletedDocs.length} documents.`));
+
+        // find references to the deleted document in content
+        console.log("Checking references...");
+        const referringFiles = [];
+        const allDocs = await Document.findAll();
+        for (const document of allDocs.iterDocs()) {
+          const rawBody = document.rawBody;
+          for (const deleted of deletedDocs) {
+            const url = `/${locale}/docs/${deleted}`;
+            if (rawBody.includes(url)) {
+              referringFiles.push(`${document.url}`);
+            }
+          }
+        }
+
+        console.warn(
+          chalk.yellow(
+            `\n${referringFiles.length} files are referring to the deleted document.`
+          )
+        );
+        if (referringFiles.length) {
+          console.warn(
+            chalk.yellow(
+              `Please update following files to remove the links:\n\t${referringFiles.join(
+                "\n\t"
+              )}`
+            )
+          );
+        }
```

Comment:
What about:
```suggestion
        if (referringFiles.length) {
          console.warn(
            chalk.yellow(
              `\n${referringFiles.length} files are referring to the deleted document.` +
              `Please update the following files to remove the links:\n\t${referringFiles.join(
                "\n\t"
               )}`
            )
          );
        } else {
          console.log(
            chalk.green("\nNo file is referring to the deleted document.");
        }
```

(Not tested and not prettified)

Reply:
Thanks! There was a syntax error so I did these changes in local and ran prettier.